index.html

```diff
@@ -0,0 +1,235 @@
+            <button type="button" class="btn-tonic">See Project</button>
+            </div>
+            </div>
+            <div class="card">
+              <img src="images/multiple.png" alt="multiple"> 
+              <div class="desc">
+           <h2>Multi-post stories</h2>
+           <div class="tonic-items-title">
+            <p class="item1" >CANOPY</p>
+           <div class="list-tonic">
+          <ul class="tonic-items">
+            <li class="dot"></li>
+           <li class="tonic-item2">Back End Dev</li>
+           <li class="dot"></li>
+           <li class="tonic-item3">2015</li>
+         </ul>
+       </div>
+       </div>
+          <p class="tonic-description">A daily selection of privately personalized reads; no accounts or
+            sign-ups required.</p>
+          <ul class="capabilities">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button type="submit" class="btn-tonic">See Project</button>
+          </div>
+          </div>
+          <div class="card">
+              <img src="images/tonic.png" alt="Tonic"> 
+              <div class="desc">
+           <h2>Tonic</h2>
+           <div class="tonic-items-title">
+            <p class="item1" >CANOPY</p>
+           <div class="list-tonic">
+          <ul class="tonic-items">
+            <li class="dot"></li>
+           <li class="tonic-item2">Back End Dev</li>
+           <li class="dot"></li>
+           <li class="tonic-item3">2015</li>
+         </ul>
+       </div>
+       </div>
+          <p class="tonic-description">A daily selection of privately personalized reads; no accounts or
+            sign-ups required.</p>
+          <ul class="capabilities">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button type="submit" class="btn-tonic">See Project</button>
+          </div>
+          </div>
+          <div class="card">
+              <img src="images/multiple2.png" alt="multiple2"> 
+              <div class="desc">
+           <h2>Multi-post stories</h2>
+           <div class="tonic-items-title">
+            <p class="item1" >CANOPY</p>
+           <div class="list-tonic">
+          <ul class="tonic-items">
+            <li class="dot"></li>
+           <li class="tonic-item2">Back End Dev</li>
+           <li class="dot"></li>
+           <li class="tonic-item3">2015</li>
+         </ul>
+       </div>
+       </div>
+          <p class="tonic-description">A daily selection of privately personalized reads; no accounts or
+            sign-ups required.</p>
+          <ul class="capabilities">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button type="button" class="btn-tonic">See Project</button>
+          </div>
+          </div>
+          </section>
+            <!--ABOUT MYSELF SECTION-->
+          <section class="about">
```

Comment:
- [ ] The background color & border-style should be the same as the original website.
- your website
![edit2](https://user-images.githubusercontent.com/26767383/149562551-9be61a44-4743-4b11-a826-e77c45ffd79f.PNG)
- Original website
![o2](https://user-images.githubusercontent.com/26767383/149563378-851138c9-23b2-4e53-9147-d2e5841b766e.PNG)